ivy/functional/frontends/paddle/tensor/tensor.py

```diff
@@ -672,6 +672,10 @@ def std(self, axis=None, unbiased=True, keepdim=False, name=None):
     def trunc(self, name=None):
         return paddle_frontend.Tensor(ivy.trunc(self._ivy_array))
 
+    @with_supported_dtypes({"2.5.1 and below": ("float32", "float64")}, "paddle")
+    def stanh(self, scale_a=0.67, scale_b=1.7159, name=None):
+        return paddle_frontend.stanh(self._ivy_array, scale_a=scale_a, scale_b=scale_b)
+
     @with_supported_dtypes(
```

Comment:
`test_paddle_stanh` is failing as well, which means the failures of the method you wrote are caused by the used function, not your code per se. 
So, we can merge the PR as soon as you resolve the lint errors highlighted in the CI checks. Thanks!

Reply:
hey @AnnaTz  i made all the changes now its ready to merge. 🚀 

thank you